docs/configuration_options.md

```diff
@@ -3604,3 +3604,147 @@ Store adaptive timeout metrics in Argus. Disabled for performance tests only.
 **default:** True
 
 **type:** boolean
+
+
+## **cloud_api_url** / SCT_CLOUD_API_URL
+
+Scylla Cloud API endpoint URL (e.g., https://app.ext.lab.scylla.cloud/api)
+
+**default:** N/A
+
+**type:** str (appendable)
+
+
+## **cloud_api_token** / SCT_CLOUD_API_TOKEN
+
+Scylla Cloud API authentication token
+
+**default:** N/A
+
+**type:** str (appendable)
+
+
+## **cloud_account_id** / SCT_CLOUD_ACCOUNT_ID
+
+Scylla Cloud account ID
+
+**default:** N/A
+
+**type:** int
+
+
+## **cloud_credentials_path** / SCT_CLOUD_CREDENTIALS_PATH
+
+Path to Scylla Cloud credentials file, if stored locally
+
+**default:** N/A
+
+**type:** str (appendable)
+
+
+## **cloud_env** / SCT_CLOUD_ENV
+
+Scylla Cloud environment (e.g., lab).<br>WARNING: environment specific credentials will be read from keystore (s3 or local file) and<br>will override 'cloud_api_url', 'cloud_api_token' and 'cloud_account_id' parameters
```

Comment:
I'd prefer removing these if are overridden anyway. Someday, someone may put api token in a wrong place which will be exposed to logs or somewhere public